python/GangaDirac/Lib/Backends/DiracBase.py

```diff
@@ -222,7 +224,11 @@ def submit(self, subjobconfig, master_input_sandbox):
         f = open(dirac_script_filename, 'w')
         f.write(dirac_script)
         f.close()
-        return self._common_submit(dirac_script_filename)
+        try:
+            return self._common_submit(dirac_script_filename)
+        except GangaDiracException as err:
+            logger.warning("Error during job submission: %s" % err)
+            raise GangaException(err)
```

Comment:
Apologies - I started the review end of last week and I guess there have been changes since then.